src/cf_cre.erl

```diff
@@ -162,7 +162,8 @@ when is_tuple( App ),
     false ->
 
       % start process
-      _Pid = spawn_link( fun() -> stage( Lam, Fa, DataDir, Mod, R, LibMap, ModState ) end ),
+      Runtime = self(),
+      _Pid = spawn_link( fun() -> stage( Runtime, Lam, Fa, DataDir, Mod, R, LibMap, ModState ) end ),
```

Comment:
you mean `apply/3` instead of `spawn_link/1`? but a call to `apply/3` would block. (And yes, you would have to export `stage/n` then.)